### DISCUSSION TITLE
Open up for group discussion: Surgery modifiers

### DISCUSSION THREAD

**Discussion by rtmill** [OHDSI/OncologyWG#486] [Ideas]
# Proposed Solutions

Dependent on the outcome of 'Procedure Relation Hierarchy' component. 

With adequate vocabulary additions and relations to qualify the specific procedure records/concepts by site, procedure type, and intent, we can potentially keep the procedure hierarchies as they are. 

Following this course, what "modifiers" would we need to implement in order to fully document the details of the procedure? 

Some examples:
- Margins
- Topography
- Outcome
- Others?

**Comment by peterprinsen-iknl** [comment]
In the Netherlands Cancer Registry we register the following details for the surgery:

* Radicality of resection (margins):
	- R0
	- R1
	- R2
	- unknown
* Surgical method (translated from Dutch so the common English terms may be different):
	- open surgery
	- conventional keyhole surgery, no conversion
	- conventional keyhole surgery, with conversion
	- robot-assisted keyhole surgery, no conversion
	- robot-assisted keyhole surgery, with conversion
	- unknown
	
We do not register the topography explicitly. It is implicit: the surgery is always on the primary tumor (the site of which can be derived from the diagnosis) unless the procedure name specifies that it was a surgery on the metastases (the location(s) of which we also register). The specific surgery concept usually also specifies the topography (prostatectomy, left hemicolectomy, etc.) although we have not always registered at this specific level.

We do have a hard time mapping the metastases surgeries. We map to concepts that do not specify that the surgery was for metastases. On the other hand, that may be ok: a partial hepatectomy for a patient who does not have a primary liver tumor is a surgery for metastases in the liver. But that may be specific to the cancer registry since we only register treatments pertaining to the cancer.

So, we could derive the topography (although it would complicate the ETL). Does it make sense to add this level of detail or could the alternative be a modifier that specifies that it was a surgery on the primary tumor, lymph nodes, or for metastases?

What kind of outcome are you thinking of?

We also register surgical complications (during the patient's stay at the hospital or within 30 days after surgery).

**Comment by cgreich** [comment]
@peterprinsen-iknl:

This is interesting.

> We do not register the topography explicitly. It is implicit: the surgery is always on the primary tumor (the site of which can be derived from the diagnosis)

Well, the standard surgery procedures are topology based, i.e., "hysterectomy". We need to think how we handle this.

> a partial hepatectomy for a patient who does not have a primary liver tumor is a surgery for metastases in the liver. But that may be specific to the cancer registry since we only register treatments pertaining to the cancer

There actually is no such a thing as a complete hepatectomy: The patient cannot live without a liver. But we do need to figure out how we distinguish between surgery on the primary and the metastases.

**Comment by rtmill** [comment]
The coverage of details contained within the procedure concept is variable depending on which vocabulary we're referring to, and really how the effort of standardizing the procedure hierarchies turns out. Additionally, there are often "attributes" of the procedure that are recorded in different fields from the coding itself, such as those listed above. For topography, I meant along the lines of "Breast excision" with a topography of "upper left quadrant"

Topography and extent of resection seem to be the most inquired about and straightforward up to this point, clear attributes of a procedure that are otherwise missed, but there are others. 

@peterprinsen-iknl Complications is a good and tricky example, in terms of OMOP representation, especially when the source data explicitly provides that relationship.  Could be post-operative complications (e.g. infarction, hematoma) or post-operative deficits (e.g. weakness, dysphasia) or others but to me it boils down to whether or not we need to preserve those relationships in the data or is including the separate records in the patient timeline sufficient enough for analysis.
